### DISCUSSION TITLE
Face swap while keeping everything in the picture the same except for the face?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [lllyasviel/Fooocus#3571] [General]
Face swap while keeping everything in the picture the same except for the face?

**Comment by Yogesh-DevHub** [comment]
How to make consistent characters is in this video 
https://youtu.be/180ovCHT-Ss?si=hp809wAGvgREteZ9  

Explained why and when you will fail in generating consistent character in this video 
https://youtu.be/ky2NcI0aCHU?si=aDRHazwn7IMzh3qL  

How can you generate consistent character all the time (training a lora) in this video.  
https://youtu.be/-L9tP7_9ejI?si=ori_LODs9clsrM6H
